cirq/linalg/decompositions.py

```diff
@@ -854,3 +857,98 @@ def kak_decomposition(unitary_object: Union[np.ndarray, 'cirq.SupportsUnitary'],
         global_phase=g * inner_cannon.global_phase,
         single_qubit_operations_before=(b1, b0),
         single_qubit_operations_after=(a1, a0))
+
+
+def kak_vector(unitary: np.ndarray):
```

Comment:
Argument type is `Union[Iterable[np.ndarray], np.ndarray]`

Reply:
Done.